contracts/BSCValidatorSet.sol

```diff
@@ -67,6 +79,15 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     uint256 incoming;
   }
 
+  // BEP-127 Temporary Maintenance
+  struct ValidatorExtra {
```

Comment:
Please move  `// BEP-127 Temporary Maintenance` inside the `ValidatorExtra ` struct

Reply:
fixed